libstatistics_collector/README.md

```diff
@@ -0,0 +1,4 @@
+# libstatistics_collector
+
+## Description
+This C++ library provides lightweight, constant time + memory
```

Comment:
Incomplete description?

Reply:
I have updated the README. Please take a look again.